website/docs/howto.md

```diff
@@ -84,6 +85,49 @@ Note the `match` field, which defines the scope of objects to which a given cons
 
 Note that if multiple matchers are specified, a resource must satisfy each top-level matcher (`kinds`, `namespaces`, etc.) to be in scope. Each top-level matcher has its own semantics for what qualifies as a match. An empty matcher is deemed to be inclusive (matches everything). Also understand `namespaces`, `excludedNamespaces`, and `namespaceSelector` will match on cluster scoped resources which are not namespaced. To avoid this adjust the `scope` to `Namespaced`.
 
+### The parameters field
+
+The `parameters` field describes the intent of a constraint as parameterized and configured by the operator. It can be referenced by the ConstraintTemplate's rego using `input.parameters` as part of the policy evaluation. Gatekeeper populates `input.parameters` with values passed into the `parameters` field in the Constraint.
+
+Example:
+```yaml
+      rego: |
```

Comment:
Should we be mentioning constraint templates in a how-to about constraints?

It may be more friendly to introduce constraints to new users first, then have a separate guide for when they find the need to write templates.

Reply:
constraint template is the first thing we introduce in the howto doc: https://open-policy-agent.github.io/gatekeeper/website/docs/howto#constraint-templates

Reply:
Hrm, might be good to move the concept of writing templates to a different section to avoid overloading new users. But looks like that's separate work.